src/core/identifyable.ts

```diff
@@ -1,4 +1,7 @@
 export interface Identifyable {
     name: string;
     id: string;
+    candyCost?: number;
```

Comment:
This does not belong here. You would actually create a new interface for this.

Reply:
That's true, I was just doing it quickly and totally ignored it, but it deserves it's own interface. This can be mandatory, but has to be optional on the other two, since not every evolution requires item or buddy distance.